bin/run.js

```diff
@@ -27,6 +27,12 @@ try {
 }
 
 // process runfile.js
-console.log('Processing runfile...')
-var runfile = require(path.resolve('./runfile'));
+try {
+    console.log('Processing runfile...')
+    var runfile = require(path.resolve('./runfile'));
```

Comment:
What if during that `require` module actually will be required but some error encounter within `runfile.js`? It will probably be also caught by that try/catch, providing not correct information. I think we need to check error type in `catch` part in that case. Display message if module not found and rethrow it for any other error.